synapse/module_api/__init__.py

```diff
@@ -401,6 +401,23 @@ def email_app_name(self) -> str:
         """
         return self._hs.config.email.email_app_name
 
+    @property
+    def server_name(self) -> str:
+        """The server name for the local homeserver.
+
+        Added in Synapse v1.53.0.
```

Comment:
Self-reminder to change this (and the one below) if it gets through quickly enough for 1.52 (unlikely since the RC cut-off is tomorrow)